Source/WebKit/NetworkProcess/webrtc/NetworkRTCUDPSocketCocoa.mm

```diff
@@ -129,6 +129,40 @@ static dispatch_queue_t udpSocketQueue()
     m_connections->sendTo(data, address, options);
 }
 
+static RTC::Network::EcnMarking getECN(nw_content_context_t nwContext)
+{
+    // Set Default ECN value
+    RTC::Network::EcnMarking ecn = RTC::Network::EcnMarking::kNotEct;
+
+    auto protocol = nw_protocol_copy_ip_definition();
+    auto metadata = adoptNS(nw_content_context_copy_protocol_metadata(nwContext, protocol));
+
+    if (metadata && nw_protocol_metadata_is_ip(metadata.get())) {
+        nw_ip_ecn_flag_t ecnFlag = nw_ip_metadata_get_ecn_flag(metadata.get());
+        switch (ecnFlag) {
+        case nw_ip_ecn_flag_non_ect:
+            ecn = RTC::Network::EcnMarking::kNotEct;
```

Comment:
It would read better as:
`return RTC::Network::EcnMarking::kNotEct;` here and below, even for the default case.

Reply:
@youennf  Thanks for the review. Updated.